docs/source/share_dataset.rst

```diff
@@ -220,29 +220,48 @@ Other files can safely be deleted.
 Uploading your files
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Once the repo is cloned, you can add the dataset script and dataset infos.
+Once the repo is cloned, If you need to add data files, instead of relying on the data to be hosted
+elsewhere, add these files using the following steps. Let's say that the files you're adding are ``*.json`` files, then:
 
-You can add these files to the staging environment and verify that they have been correctly staged with the ``git
-status`` command:
+.. code-block:: bash
+
+    cp /somewhere/data/*json .
+    git lfs track *.json
+    git add .gitattributes
+    git commit -m "track json files"
+    git add *json

```

Comment:
```suggestion
    cp /somewhere/data/*.json .
    git lfs track *.json
    git add .gitattributes
    git commit -m "track json files"
    git add *.json
```
*.json instead of *json